test/dlc_tests/eks/pytorch/training/test_eks_pytorch_training.py

```diff
@@ -104,16 +104,14 @@ def test_eks_pytorch_single_node_training(pytorch_training):
 )
 @pytest.mark.model("resnet18")
 @pytest.mark.integration("pt_s3_plugin")
-def test_eks_pt_s3_plugin_single_node_training(pytorch_training, pt17_and_above_only):
+def test_eks_pt_s3_plugin_single_node_training(pytorch_training, outside_versions_skip):
```

Comment:
Same here. is `outside_versions_skip` declared in the test function definition work? i think it is redundant